dsgrid/config/time_dimension_base_config.py

```diff
@@ -254,26 +243,34 @@ def _align_time_interval_type(self, df, project_time_dim):
         time_col = time_col[0]
 
         if dtime_interval == ptime_interval:
-            return df
+            return df, False
```

Comment:
Since above you just call `self._handle_time_difference(df, project_time_dim, wrap_time)` in this case, could you do that here instead and avoid having to pass a tuple of (dataframe, bool)?